### PR TITLE
Moving of Props

### PR DESCRIPTION
**Pull request by SticksDeveloper**

### CODE REVIEW

**Comment by SticksDeveloper** [comment]
also ignore the you're gonna hate me for this desorro everything in that got changed back

File: build.gradle [hunk 1]
```diff
@@ -30,7 +30,7 @@ tasks.withType(JavaCompile).configureEach {
 minecraft {
     mcVersion = '1.12.2'
 
-    username = "Desoroxxx"
+    username = "Dev"
```
Comment (Desoroxxx) [inline build.gradle:33]: Why, dev doesn't even have a skin

Comment (Desoroxxx) [reply build.gradle:33]: / Please change it to your username but don't push as change

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

**Comment by Desoroxxx** [comment]
Anyway won't merge this since the props mod is a thing

File: build.gradle (hunk 1 above)
Comment (SticksDeveloper) [reply build.gradle:33]: > / Please change it to your username but don't push as change

That's not my username...

Comment (SticksDeveloper) [reply build.gradle:33]: > Why, dev doesn't even have a skin

Great question you'd think for a name like that he'd at least have a skin

**Comment by SticksDeveloper** [comment]
> Anyway won't merge this since the props mod is a thing

Wdym?

**Comment by Desoroxxx** [comment]
> > Anyway won't merge this since the props mod is a thing
> 
> Wdym?

Missphrased that, this won't be merged until the props mod is a thing

File: build.gradle (hunk 1 above)
Comment (Desoroxxx) [reply build.gradle:33]: > > / Please change it to your username but don't push as change
> 
> That's not my username...

Yeah, that's the problem, you should set this as your username if you care about it, but don't push that, or just leave it like that.

**Comment by SticksDeveloper** [comment]
> > > Anyway won't merge this since the props mod is a thing
> > 
> > 
> > Wdym?
> 
> Missphrased that, this won't be merged until the props mod is a thing

oh I see thank you for clearing up I'll draft this

**Pull request closed without merging**